TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousBase.java

```diff
@@ -75,7 +75,7 @@ public abstract class AutonomousBase extends LinearOpMode {
 
     boolean     blueAlliance    = true;  // Is alliance BLUE (true) or RED (false)?
     boolean     forceAlliance   = false; // Override vision pipeline? (toggled during init phase of autonomous)
-    int         fiveStackHeight = 5;     // Number of cones remaining on the 5-stack (always starts at 5)
+    static int         fiveStackHeight = 5;     // Number of cones remaining on the 5-stack (always starts at 5)
```

Comment:
This static should probably be in hardware class. Adding it to autonomous base is going to create a dependency of auto from telelop. Gotta push this data down into a shared class that both teleop and auto use.